File: python/legacy_intro/legacyintro.py
class InvoiceItemManager:
    def save_item_total_amount(self, quantity: float, price: float, tax_rate: float) -> None:
        total = quantity * price
        tax = total * tax_rate
        total_with_tax = total + tax
        
        with open("invoice_items.csv", "a") as f:
            f.write(f"{quantity};{price};{tax_rate};{total_with_tax}\n")
            

File: javascript/legacy_intro/legacyintro.ts
import { writeFileSync } from 'fs';

export class InvoiceItemManager {
    saveItemTotalAmount(quantity: number, price: number, taxRate: number) {
        const total = quantity * price
        const tax = total * taxRate
        const totalWithTax = total + tax
        
        writeFileSync('invoice_items.csv', `${quantity};${price};${taxRate};${totalWithTax}\n`, { flag: 'a' });
    }
}

